### ISSUE TITLE
Please provide feature Timezone select for vm select

### ISSUE THREAD

**Issue by zharymv** [VictoriaMetrics/VictoriaMetrics#3583]
Hi there!
Here is [TimeZone feature request for VM Standalone](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/3075)
thanks a lot for it, but this time please provide the same feature for vmselect.

Thanks for your time.

**Comment by hagen1778** [comment]
Hello @zharymv !
vmselect and vmsingle are using the same source code of vmui. So there is no need in adding the feature twice.

Are you using vmselect of version 1.85+ and not seeing the time zone selector in vmui?

**Comment by zharymv** [comment]
well, we'd tried version you provided, but i didn't get this option in vmselect.
let me try again.. just a moment

**Comment by zharymv** [comment]
yep, it there.
thanks a lot)
